packages/@aws-cdk/core/test/secret-value.test.ts

```diff
@@ -47,6 +47,11 @@ nodeunitShim({
     test.done();
   },
 
+  'secretsManager with versionStage and versionId'(test: Test) {
+    test.throws(() => SecretValue.secretsManager('secret-id', { versionStage: 'version-stage', versionId: 'version-id' }), /were both provided but only one is allowed/);
```

Comment:
Let's re-format this:

```suggestion
    test.throws(() => {
      SecretValue.secretsManager('secret-id', {
        versionStage: 'version-stage',
        versionId: 'version-id',
      });
    }, /were both provided but only one is allowed/);
```